workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -203,14 +191,15 @@ def make_zip64_archive(zip_filename, directory):
 
     with zip_context_manager as zip_file:
         path = os.path.normpath(directory)
-        zip_file.write(path)
+        ctx.logger.debug('Creating zip archive of: {0}'.format(path))
+        len_base_dir = len(path)
         for dirpath, dirnames, filenames in os.walk(directory):
             for dirname in sorted(dirnames):
                 path = os.path.normpath(os.path.join(dirpath, dirname))
-                zip_file.write(path)
+                zip_file.write(path, path[len_base_dir:])
```

Comment:
What about using `os.path.relpath(path, base_dir)` instead of `path[len_base_dir:]`. The result should be the same, but I believe the first option is more readable.